### ISSUE TITLE
How can data be serialized?

### ISSUE THREAD

**Issue by MartinThoma** [karpathy/convnetjs#21]
Can a network that is already trained be serialized? In which format can it be serialized?
Can I train a network with other tools and let convnetjs only read it?

**Comment by torrmal** [comment]
Hi Andrej,

When you said that converted alexnet from Caffe, what made it messy?  are there particular things that caffee does in alexnet that currently convnetjs doesn't support?

Jorge

**Comment by karpathy** [comment]
Yeah the more annoying one was group convolution (the parameter group:2).
That's the point where I gave up because I was too lazy to implement it,
although it probably would have been not too difficult.

In particular, the AlexNet is broken down into two streams due to
historical reasons, since there were difficulties in fitting the full model
on a single GPU. So with K filters looking at input volume of depth D, the
first K/2 look at the first D/2 activation maps, and the other K/2 look at
the other D/2 activation maps.


Andrej
